nginx/ng/certificates.sls

```diff
@@ -1,6 +1,23 @@
 include:
   - nginx.ng.service
 
+{% if salt.pillar.get('nginx:ng:dh_contents') %}
+create_nginx_dhparam_key:
+  file.managed:
+    - name: /etc/nginx/ssl/dhparam.pem
```

Comment:
there's a configuration value available for `/etc/nginx`: `{% set conf_dir = nginx.get('conf_dir', '/etc/nginx') -%}`, you should use that

you might also allow them to specify the ssl directory, they may want it in `/etc/ssl`